CaloOnlineTools/HcalOnlineDb/src/HcalLutManager.cc

```diff
@@ -1757,29 +1768,37 @@ std::map<int, std::shared_ptr<LutXml>> HcalLutManager::getZdcLutXml(std::string
       _cfg.formatrevision = "1";  //???
       _cfg.generalizedindex = 0;
 
-      //HcalZDCDetId _detid(row->zdc_section, (row->zdc_zside>0), row->zdc_channel);
+      HcalZDCDetId::Section section = HcalZDCDetId::Unknown;
+      if (row->zdc_section == "ZDC EM") {
+        section = HcalZDCDetId::EM;
+        _cfg.iphi = 1;
+      } else if (row->zdc_section == "ZDC HAD") {
+        section = HcalZDCDetId::HAD;
+        _cfg.iphi = 2;
+      } else if (row->zdc_section == "ZDC LUM") {
+        continue;
+        section = HcalZDCDetId::LUM;
+      } else if (row->zdc_section == "ZDC RPD") {
+        continue;
+        section = HcalZDCDetId::RPD;
```

Comment:
Is this line ever accessed?

Reply:
No, this line is never accessed. I can delete it

section = HcalZDCDetId::RPD;